x/wasm/keeper/api.go

```diff
@@ -25,13 +28,35 @@ func (a cosmwasmAPIImpl) canonicalAddress(human string) ([]byte, uint64, error)
 	bz, err := sdk.AccAddressToBytes(human)
 	return bz, 4 * a.gasMultiplier, err
 }
+func (a cosmwasmAPIImpl) GetContractEnv(contractAddrStr string) (*wasmvm.Cache, wasmvm.KVStore, wasmvm.Querier, wasmvm.GasMeter, []byte, uint64, error) {
+	contractAddr := sdk.AccAddress(contractAddrStr)
+	_, codeInfo, prefixStore, err := a.keeper.contractInstance(*a.ctx, contractAddr)
+	if err != nil {
+		return nil, nil, nil, nil, wasmvm.Checksum{}, 0, err
+	}
+
+	cache := a.keeper.wasmVM.GetCache()
+	if cache == nil {
+		panic("cannot found instance cache")
+	}
+
+	// prepare querier
+	querier := NewQueryHandler(*a.ctx, a.keeper.wasmVMQueryHandler, contractAddr, a.gasMultiplier)
+	gas := 20 * a.gasMultiplier
```

Comment:
Why is the gas `20 * a.gasMultiplier`?
How did you calculate this number?
And hard cording is not problem?

Reply:
It has the same format as other existing above APIs above 0.14.0 .
https://github.com/line/lbm-sdk/blob/469d3e1e38a7265a1126e96954de68b6df650abe/x/wasm/keeper/api.go#L18
In 0.16.3 it's a bit cleaner with const. This will change later when the main branch is merged.

20 is the gas value I randomly set. An appropriate gas value should be determined by further testing.